accumulo-store/src/main/java/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -272,8 +273,15 @@ public Properties getPropertiesFromColumnVisibility(final String group, final by
             if (null != propertyDef) {
                 final Serialisation serialiser = propertyDef.getSerialiser();
                 try {
-                    properties.put(schema.getVisibilityProperty(),
-                            serialiser.deserialise(columnVisibility));
+                    if (columnVisibility == null || columnVisibility.length == 0) {
+                        if (serialiser.deserialiseEmptyBytes() != null) {
```

Comment:
Perhaps store the response from serialiser.deserialiseEmptyBytes() so we don't have to call it twice.